docs/installation/Setup-OpenFunction-Builder-with-Tekton.md

```diff
@@ -70,7 +70,7 @@ If you want to use ```NodePort``` to expose the Tekton dashboard service, you ne
 ### Install Tekton pipeline
 
 ```bash
-kubectl apply --filename https://openfunction.sh1a.qingstor.com/tekton/pipeline/v0.23.0/release.yaml
+kubectl apply --filename https://openfunction.sh1a.qingstor.com/tekton/pipeline/latest/release.yaml
```

Comment:
We'd better use a fix version instead of latest

Reply:
Ok. Let me restore it.